qutebrowser/mainwindow/tabwidget.py

```diff
@@ -172,6 +172,7 @@ def get_tab_fields(self, idx):
         fields['perc_raw'] = tab.progress()
         fields['backend'] = objects.backend.name
         fields['private'] = ' [Private Mode] ' if tab.private else ''
+        fields['muted'] = '[M] ' if tab.is_muted() else ''
```

Comment:
I'm not sure if `[M] ` is the best indicator. I considered a unicode speaker, but that won't show for everyone if they don't have fonts properly set up. Suggestions are appreciated..